contrib/icu-cmake/CMakeLists.txt

```diff
@@ -15,329 +15,245 @@ set(ICUDATA_SOURCE_DIR "${ClickHouse_SOURCE_DIR}/contrib/icudata/")
 # These lists of sources were generated from build log of the original ICU build system (configure + make).
```

Comment:
Btw for me was a lot of easier to just copy file names from https://github.com/unicode-org/icu/blob/main/icu4c/source/common/sources.txt and https://github.com/unicode-org/icu/blob/main/icu4c/source/i18n/sources.txt